src/app/analyze/analyze.test.mockdata.js

```diff
@@ -0,0 +1,34 @@
+export const mockFileResults = {
+    oneFileLarger: [
+        {
+            size: 125,
+            baseBranchSize: 100,
+        },
+    ],
+    oneFileSmaller: [
+        {
+            size: 200,
```

Comment:
This file is larger?

Reply:
The mock new size is bigger than the mock base branch size

Reply:
This test case is the same as `oneFileLarger` no? size > baseBranchSize.
Should size be smaller than baseBranchSize here?